### PR TITLE
[DOCS] Adding a few resources to resources section

### PR DESCRIPTION
**Pull request by franzihei**
Adding:
- Foundry
- Scaffold-ETH
- Docs community translations repo
- Sourcify

### CODE REVIEW

File: docs/resources.rst [hunk 1]
```diff
@@ -112,6 +115,9 @@ Solidity Tools
 * `PIET <https://piet.slock.it/>`_
     A tool to develop, audit and use Solidity smart contracts through a simple graphical interface.
 
+* `Scaffold-ETH <https://github.com/scaffold-eth/scaffold-eth>`
```
Comment (cameel) [inline docs/resources.rst]:
```suggestion
* `Scaffold-ETH <https://github.com/scaffold-eth/scaffold-eth>`_
```

**Review by cameel — COMMENTED** [review]
Looks good, just needs a tiny syntax correction.

File: docs/resources.rst [hunk 2]
```diff
@@ -112,6 +115,9 @@ Solidity Tools
...
+    Forkable Ethereum development stack focused on fast product iterations.
+
```
Comment (cameel) [inline docs/resources.rst]: Also, looks like we already have it listed (but in the IDE section, which seems wrong).

**Comment by ekpyron** [comment]
Overlaps with https://github.com/ethereum/solidity/pull/12596 by the way.

File: docs/resources.rst (hunk 1 above)
Comment (cameel) [reply docs/resources.rst]: Fixed.

File: docs/resources.rst (hunk 2 above)
Comment (cameel) [reply docs/resources.rst]: I removed it from the IDE section.

**Review by cameel — APPROVED** [review]
I fixed the two small issues I listed and I think it's ready to merge now.

**Pull request merged**